assets/src/blocks/Media/MediaEditor.js

```diff
@@ -1,15 +1,14 @@
-import {Fragment, useCallback} from '@wordpress/element';
-import {PanelBody, TextControl} from '@wordpress/components';
-import {MediaPlaceholder, InspectorControls, RichText} from '@wordpress/block-editor';
-import {debounce} from '@wordpress/compose';
-
 import {MediaElementVideo} from './MediaElementVideo';
-import {useSelect} from '@wordpress/data';
 import {lacksAttributes} from './MediaBlock';
 
+const {useSelect} = wp.data;
+const {MediaPlaceholder, InspectorControls, RichText} = wp.blockEditor;
 const {__} = wp.i18n;
 const {apiFetch} = wp;
-const {addQueryArgs} = wp.url;
+const {debounce} = wp.compose;
+const {PanelBody, TextControl} = wp.components;
+const {url: {addQueryArgs}} = wp.url;
```

Comment:
This results in the following console error in the editor: 
```
Uncaught TypeError: Cannot read properties of undefined (reading 'addQueryArgs')
```

Reply:
I see in other places you've left it as `const {addQueryArgs} = wp.url;` and I think it should work fine 🤔

Reply:
Oh my fault!

That is because `const {url: {addQueryArgs}} = wp.url;` should be converted to `const {url: {addQueryArgs}} = wp;` in that case since you're taking the `url` field from `wp`. 

And `const {addQueryArgs} = wp.url;` you are directly destructuring `wp.url`

Reply:
`url` does not exist in `wp.url` but yes into `wp`